README.md

```diff
@@ -1888,6 +1907,74 @@ Profile name of the Encapsulation. Valid values are String only.
 ##### `dot1q_map`
 The encapsulation profile dot1q vlan-to-vni mapping. Valid values are an array of [vlans, vnis] pairs.
 
+--
+### Type: cisco_evpn_multisite
+
+Manages Cisco Ethernet Virtual Private Network (EVPN) Multisite configurations of a Cisco device.
+
+| Platform | OS Minimum Version | Module Minimum Version |
+|----------|:------------------:|:----------------------:|
+| N9k      | 7.0(3)I7(2)        | 1.9.0                  |

```

Comment:
Were there changes between `7.0(3)I7(1) and 7.0(3)I7(2)`?

Reply:
Tested with I7(1) and it works. Updated these to say `7.0(3)I7(1)` now